MonoGame.Framework.Native.sln

```diff
@@ -13,6 +13,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MonoGame.Tests.WindowsVK",
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "desktopvk", "native\monogame\desktopvk.vcxproj", "{60D6243F-CC40-D9B5-157F-8A5B8128B70A}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "windowsdx", "native\monogame\windowsdx.vcxproj", "{CC33DB03-389E-8F7A-81DC-4020ED856DCF}"

```

Comment:
Should the native project be called WindowsDX like i did here?  Or maybe DestkopDX to match DesktopVK?

I used WindowsDX because there is no Mac or Linux support... which was the intent of the term "desktop".

Thoughts?

Reply:
Maybe match the platform definitions? desktopVK, windowsGDK...

Reply:
Ok... so GDK... are we really GDK?

In this case we're just using SDL2 (window and inputs) + DirectX12 + no audio yet...  so is this considered GDK?

Or is that just plain Windows ?